bims/static/js/views/taxon_detail.js

```diff
@@ -26,6 +26,9 @@ define(['backbone', 'ol', 'shared'], function (Backbone, ol, Shared) {
             return template(data);
         },
         showDetail: function (name) {
+            if(isHealthyrivers){
```

Comment:
no need conditions for this, because already changed on healthyrivers.html

Reply:
hi @meomancer , that condition is so we can switch the side panel when clicking on site and taxon, as shown below:
![peek 2018-08-20 14-49](https://user-images.githubusercontent.com/26101337/44327879-23ed5a80-a48a-11e8-9b11-5df11256aceb.gif)